.markdownlint.json

```
@@ -2,7 +2,7 @@
   "default": true,
   "first-line-heading": false,
   "line-length": false,
-  "list-marker-space": { "ol_multi": 2, "ol_single": 2 },
+  "list-marker-space": { "ol_multi": 2, "ol_single": 2, "ul_multi": 2, "ul_single": 2 },
```

There seem to be issues with setting them to `2`. What do you think about `1`? It feels more natural to me anyway. Typing two spaces for me automatically inserts a `.` (dot).
```suggestion
  "list-marker-space": {
    "ol_multi": 1,
    "ol_single": 1,
    "ul_multi": 1,
    "ul_single": 1
  },
```